ts/screens/onboarding/OnboardingNotificationsInfoScreenConsent.tsx

```diff
@@ -138,17 +203,16 @@ const OnboardingNotificationsInfoScreenConsent = () => {
           title={I18n.t("onboarding.infoConsent.title")}
           subtitle={I18n.t("onboarding.infoConsent.subTitle")}
         >
-          <View style={styles.container}>
-            <InstructionRow
-              icon={<NotificationsIcon width={44} height={44} />}
-              title={I18n.t("onboarding.infoConsent.instruction1")}
-            />
-            <View style={styles.separator} />
-            <InstructionRow
-              icon={<NotificationsToggleIcon width={44} height={23} />}
-              title={I18n.t("onboarding.infoConsent.instruction2")}
+          <ContentWrapper>
+            <FlatList
+              data={instructions}
+              renderItem={renderItem}
+              keyExtractor={keyExtractor}
```

Comment:
Do we need this? Doc seems to say that it looks either for a `key` or and `id` property on the `item` instance and then it falls back to using the `index`. If we are just generating a string with the `index`, cannot we just ignore this property and let it use its default behaviour?